lib/minitest/fail_fast_plugin.rb

```diff
@@ -24,7 +24,7 @@ def self.fail_fast?
     end
 
     def record result
-      if result.failures.any?
+      if result.failures.reject{|failure| failure.kind_of?(Minitest::Skip)}.any?
```

Comment:
Could we add spaces around the curly-braces? So it would look like:
`result.failures.reject { |failure| failure.kind_of?(Minitest::Skip) }.any?`